README.md

```diff
@@ -11,19 +11,18 @@ A fast implementation of the Aho-Corasick algorithm using the compact double-arr
 
 ## Overview
 
-Daachorse is a crate for fast multiple pattern matching using
-the [Aho-Corasick algorithm](https://dl.acm.org/doi/10.1145/360825.360855),
-running in linear time over the length of the input text.
-For time- and memory-efficiency, the pattern match automaton is implemented using
-the [compact double-array data structure](https://doi.org/10.1016/j.ipm.2006.04.004).
-The data structure not only supports constant-time state-to-state traversal,
-but also represents each state in a compact space of only 12 bytes.
-
-For example, compared to the NFA of the [aho-corasick](https://github.com/BurntSushi/aho-corasick) crate
-that is the most poplar Aho-Corasick implementation in Rust,
-Daachorse can perform pattern matching **3.3–5.4 times faster**
-while consuming **56–60% smaller** memory, when using a word dictionary of 675K patterns.
-Other experimental results can be found in
+Daachorse is a crate for fast multiple pattern matching using the
+[Aho-Corasick algorithm](https://dl.acm.org/doi/10.1145/360825.360855), running in linear time over
+the length of the input text. This crate uses the
+[compact double-array data structure](https://doi.org/10.1016/j.ipm.2006.04.004) for implementing
+the pattern match automaton for time and memory efficiency. The data structure not only supports
+constant-time state-to-state traversal but also represents each state in a tight space of only 12
```

Comment:
```suggestion
constant-time state-to-state traversal but also represents each state in a space of only 12
```
I don't think "tight" is often used in such sentences. In [Hyper Collocation](https://hypcol.marutank.net/), "tight space" hits only 166 occurrences.

I suggest eliminating "tight" because "only 12 bytes" has already indicated it.